### DISCUSSION TITLE
is replit still available under github student developer program?

### DISCUSSION THREAD

**Discussion by Abhinav-Kumar012** [community/community#41964] [GitHub Education] [answered]
I saw the replit offer a few days ago but can seem to find it now as I want to apply for it now .It's not visible in the offers section.

**Comment by jfullstackdev** [comment]
`replit` in itself has the free version unless you want more like the paid version

**Reply by TheEthicalApproach** [reply]
It would be extremely helpful to gain some assistance with replit as I'm dealing with half a phone at the moment. Android 11 running on seemingly 12 IOS with Darken/blablabla all over the place and I'm not at my wits end yet but I just went through a data cash dump marathon with someone I've reported know as Tortel (misuse of python code assist software) running on either windows 10 or Mac OS laptops with even the audacity to forward a few screenmirror_Apks Emulators and SD Admin stalemate bootloader coding. 

If you'd like I can forward some of the complied data and sys traces (12 or 13 maybe more traces )



Much appreciated 


- TheEthicalApproach

**Reply by Musthuakqha** [reply]
Hi

**Comment by dev-737** [comment]
Would also like to know. As I also am trying to redeem the replit hacker plan that had an offer for 3 months free, but it has seemingly vanished from the list of offers available. Is it no longer available now?

**Reply by TheEthicalApproach** [reply]
Here here!!!

**Comment by mishra-prajjawal** [comment]
@Abhinav-Kumar012  
Try the replit website 
https://replit.com/site/github-students 
might work.

**Reply by antgame11** [reply]
nope

**Reply by mishra-prajjawal** [reply]
worked for me pal

**Reply by dev-737** [reply]
> worked for me pal

Guess you need a new repl account or an account without any previous subscriptions then. It didn't work for me :(

**Comment by Abhinav-Kumar012** [comment, marked as answer]
looks like replit has changed its pricing model
